runtime/shibuya/src/lib.rs

```diff
@@ -1929,3 +1937,256 @@ cumulus_pallet_parachain_system::register_validate_block! {
     BlockExecutor = cumulus_pallet_aura_ext::BlockExecutor::<Runtime, Executive>,
     CheckInherents = CheckInherents,
 }
+#[cfg(test)]
```

Comment:
Nit-picky: a new line would be nice.

For the follow-up, I'd suggest creating an item to see how to build up on these runtime tests.
I'm not sure we should mock _everything_, this seems more appropriate for _semi-live_ testing via RPC, as @shaunxw already suggested.